config/setupTests.js

```diff
@@ -1,33 +1,36 @@
+const useChrome = () => ({
```

Comment:
Maybe it is time to push this somewhere and reuse it in apps. We have this in almost all apps with different definitions/implementations.